lib/addons/amp/index.ts

```diff
@@ -78,6 +80,8 @@ export class AmpAddOn implements ClusterAddOn {
     }
 
     @dependable(AdotCollectorAddOn.name)
+    @dependable(KubeStateMetricsAddOn.name)
```

Comment:
since it worked before without it, why are we enforcing this as a hard dependency now? This means no one will be able to deploy AMP without kubestate metrics and prometheus exporter, blueprint will just fail. Anyone who used AMP add-on before will get failures now for stacks that used to work. 
If it is an optional dependency let's discuss the path.

Reply:
Fixed